randovania/games/prime1/logic_database/Chozo Ruins.txt

```diff
@@ -1886,55 +1903,97 @@ Extra - aabb: [225.01727, 76.59607, 54.75615, 343.10974, 209.55296, 138.8626]
 > In Arena (Front); Heals? False; Spawn Point; Default Node
   * Layers: default
   * Extra - world_position: (285.036987, 83.419861, 66.368805)
-  > Door to Sun Tower Access
-      After Flaahgra or After Chozo Ghosts (Sunchamber)
   > Door to Sunchamber Access
-      After Chozo Ghosts (Sunchamber)
+      After Flaahgra and After Chozo Ghosts (Sunchamber)
   > Event - Flaahgra
       Any of the following:
           All of the following:
-              Morph Ball
-              Any of the following:
-                  # Requirements for destroying the bomb slots
-                  Can Use Bombs
-                  Power Bomb ≥ 4 and Combat (Beginner) and Can Use Power Bombs
-              Any of the following:
-                  # Requirements for getting to the bomb slots
-                  Shoot Power Beam
-                  Missile ≥ 10 and Shoot Any Beam
-                  Space Jump Boots and Combat (Beginner) and L-Jump (Beginner)
-                  Combat (Intermediate) and Combat/Scan Dash (Advanced)
+              # Normal Fight
+              Morph Ball and Has Any Visor
               Any of the following:
                   # Requirements for shooting the first dish
                   Shoot Power Beam
-                  Missile and Shoot Any Beam
+                  Missile and Combat (Beginner) and Shoot Any Beam
+              Any of the following:
+                  # Requirements for reaching the Bomb Slots
+                  All of the following:
+                      # Shoot down the dishes to make the tentacles retract
+                      Any of the following:
+                          Shoot Power Beam
+                          All of the following:
+                              # Shoot dishes with only Missiles
+                              Combat (Beginner) and Shoot Any Beam
+                              Any of the following:
+                                  Missile ≥ 10
+                                  Missile ≥ 4 and Combat (Advanced)
+                      Any of the following:
+                          # Health requirements
+                          Combat (Intermediate) or Normal Damage ≥ 80
+                          Combat (Beginner) and Normal Damage ≥ 25
```

Comment:
or Combat Advanced damageless

Reply:
Just in case, does that mean "change the damagless from combat intermediate to combat advanced" or am i to interpret that as "toaster didn't see the damageless combat intermediate"?

Reply:
"toaster didn't see the damageless combat intermediate"

fuck that `or` thing in an `Any` block gets me every time